CMakeLists.txt

```diff
@@ -153,6 +153,9 @@ elseif(CMAKE_C_COMPILER_ID MATCHES "MSVC")
         /wd4820 # Disable: bytes padding added after data member
         /wd4996 # Disable: potentially unsafe stdlib methods
         /wd5045 # Disable: Compiler will insert Spectre mitigation for memory load if /Qspectre switch specified
+        # See https://github.com/AOMediaCodec/libavif/pull/438
```

Comment:
Instead of referring to this pull request, could you replace this comment with an explanation? You can just edit and convert the following explanation of yours to a comment:

> I found a good way to solve this: add /source-charset:.65001 /execution-charset:.20127, it means read the source as UTF-8, but assume a console that can only print ASCII. When encountering a non-ASCII character in string, a C4556 (character represented by universal-character-name 'char' cannot be represented in the current code page (20127)) will be generated, but comment usage will be ignored silently.